r/R/array.R

```diff
@@ -216,6 +216,49 @@ Array$create <- function(x, type = NULL) {
 #' @include arrowExports.R
 Array$import_from_c <- ImportArray
 
+
+#' Concatenate zero or more Arrays
+#'
+#' Concatenates zero or more [Array] objects into a single
+#' array. This operation will copy its input; if you need
+#' the behavior of a single Array but don't need a
```

Comment:
```suggestion
#' array. This operation will make a copy of its input; if you need
#' the behavior of a single Array but don't need a
```

this is minor, but I think makes it a tiny bit clearer that it's doing a copying action (we might even go stronger?)